qdrant-landing/content/documentation/quick-start.md

```diff
@@ -4,47 +4,28 @@ weight: 11
 aliases:
   - quick_start
 ---
-
 # Quickstart
 
-## Installation
-
-The easiest way to use Qdrant is to run a pre-built image. To do this, make sure Docker is installed on your system.
-
-Download image from [DockerHub](https://hub.docker.com/r/qdrant/qdrant):
-
-```bash
-docker pull qdrant/qdrant
-```
-
-And run the service inside the docker:
-
-```bash
-docker run -p 6333:6333 \
-    -v $(pwd)/qdrant_storage:/qdrant/storage \
-    qdrant/qdrant
-```
+## Recommended Workflow
 
-In this case Qdrant will use default configuration and store all data under `./qdrant_storage` directory.
+There are two ways of using Qdrant locally: [1) Local Mode](#option-1-local-mode) and [2) Docker Mode](#option-2-docker-mode). We recommend that you first try Local Mode in [Qdrant Client](https://github.com/qdrant/qdrant-client), as it only takes a few moments to get setup. Then, you may further experiment with Qdrant Docker containers. When you are more comfortable with Qdrant, then you should deploy your app to a Free Tier [Qdrant Cloud](../cloud/quickstart-cloud/) cluster.
```

Comment:
Please start with Docker mode. It is more general, while it seems like we only have instruction for Python in local mode. 

We should change the order down below as well.

Reply:
In what situations do we want people to use local mode?